composer/blog/gcp-tech-blog/data-orchestration-with-composer/dags/data_orchestration_blog_sample_dag_test.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from airflow import models
+import internal_unit_testing
+import pytest
+
+# user should substitute their project ID
+PROJECT_ID = 'rds-sandbox'
+BUCKET_NAME = 'rds_demo_data'
+FILE_NAME = 'incoming_100.txt'
```

Comment:
Where is this test data from? If we have the licensing for it, it needs to be included. If the licensing is questionable, you can always derive a text file from the public datasets, or something else publicly available. LMK if you want help with that.

Reply:
so I've changed this -  to point to 'gs://cloud-samples-data/composer/data-orchestration-blog-example'
I'll need to create the folder and upload the file. The data itself is just scraped from an existing bq public dataset.
If it's more appropriate, I can just host this data in my own public bucket ?